recipe/meta.yaml

```diff
@@ -71,8 +75,8 @@ extra:
 # Package: dggridR
 # Type: Package
 # Title: Discrete Global Grids
-# Version: 2.0.4
-# Date: 2020-04-29
+# Version: 3.0.0
+# Date: 2022-03-17
```

Comment:
I suggest deleting the entire comment section. This is mostly for `staged-recipes` reviewers and unnecessary to update.